NAMESPACE

```diff
@@ -60,6 +60,9 @@ export(ui_todo)
 export(ui_value)
 export(ui_warn)
 export(ui_yeah)
+export(use_action_check_full)
+export(use_action_check_release)
+export(use_action_pr_commands)
```

Comment:
Maybe it's too long, but should these be `use_github_action_*()` for consistency and auto-complete predictability?

Reply:
I don't think so, that is way too long IMHO, I think `use_action` is clear